src/Routes.js

```diff
@@ -43,16 +44,19 @@ const  MalwareRoutes = () => {
         }
     ]);
 
-    return !hasSystems ?
-        (<AsynComponent
+    return (
+        <AsynComponent
```

Comment:
While we're at it, could we please fix the typo `AsynComponent` ->`AsyncComponent` to avoid future problems?